fanficfare/defaults.ini

```diff
@@ -2197,6 +2197,11 @@ include_in_category:tags
 ## chapter_start to remove the standard chapter title, as shown below.
 inject_chapter_title:false
 
+## This website removes certain HTML tags if the user is not
+## subscribed to the story. This is done by default. If you'd rather
+## not be automatically subbed to stories, change this to false.
+auto_sub:true
```

Comment:
I'm very uncomfortable with this being on by default since it will silently change the user's settings on the site.

Reply:
New settings also need to be added to `calibre-plugin/plugin-defaults.ini` and `fanficfare/configurable.py`  (search for the two places `inject_chapter_title` exists).

Reply:
> How big an issue is this? Can you provide a few example story URLs that demonstrate it?

Here's a random one: https://www.asianfanfics.com/story/view/1191573/1/gold-dust - the em tags are missing from the non-subbed json file, for instance. When the entire thing is downloaded, the size difference is approx. 150kB vs 350kB - it seems that quite a lot of stuff is missing from the unsubscribed json. 

> I hesitate to include code that changes changes the user's site settings. Plus, I have some comments below about how you've implemented it.

Yeah, it's definitely not ideal. My reasoning was that the user would most likely think it's a bug in FanFicFare moreso than a settings toggle. But I suppose it would be too invasive for it to be enabled by default. Is there any way to warn about this e.g. in Calibre when attempting to download a story when not logged in/not subscribed?

> Which JSON does this effect? And can you detect if the user is already subscribed? Is it feasible to subscribe (if not already), fetch that JSON, then unsubscribe?

The JSON URL served by the website changes depending on whether the user is not logged in (no swear words) vs logged in (swear words, lack of styling) vs logged in and subscribed (full intended text). That seems like a good compromise - I'll try implementing that and see if that works as intended.